chase/frontend/components/login/password.tsx

```diff
@@ -97,19 +98,19 @@ export default function usernameLogin({ changeLoginState }) {
               onKeyDown={(e) => e.stopPropagation()}
             >
               Nutzungsbedingungen
-            </a>
+            </Link>
             . Außerdem bin ich damit einverstanden, dass diese Website Cookies
             verwendet. Mehr Informationen dazu finden Sie in unseren{" "}
-            <a
-              href="datenschutz"
+            <Link
+              href="privacy"
```

Comment:
Careful: If you put `privacy` here instead of `/privacy` this will route relative to the current URL, e.g. to `http://localhost:3000/login/privacy` instead of `http://localhost:3000/privacy`

Reply:
Right! Was just a Placeholder since it might as well be a dmun.de link in the future, but i'll correct it anyway.